doc/maintaining/installing/install-from-docker-compose.rst

```diff
@@ -234,7 +234,7 @@ Now the datastore API should return content when visiting::
 -------------------------
 With all images up and running, create the CKAN admin user (johndoe in this example)::
 
-    docker exec -ti ckan /usr/local/bin/ckan -c /etc/ckan/production.ini sysadmin add johndoe
+    docker exec -ti ckan /usr/local/bin/ckan -c /etc/ckan/production.ini sysadmin add johndoe email=johndoe@localhost name=johndoe
```

Comment:
Yes `name=johndoe` is **not** needed. I just tested this. It should be taken out of this command in the documentation as it's confusing. As far as including the password in the command ie: `password=test1234` this could be included as well but I think it may be slightly safer to make it interactive (as it has to be entered twice)

Reply:
> Yes `name=johndoe` is **not** needed. I just tested this. It should be taken out of this command in the documentation as it's confusing. As far as including the password in the command ie: `password=test1234` this could be included as well but I think it may be slightly safer to make it interactive (as it has to be entered twice)

@Gauravp-NEC , Based on your comment. I will submit the PR.

Reply:
@Gauravp-NEC - OK thanks, please do